.github/workflows/npm_publish.yml

```diff
@@ -14,19 +14,15 @@ jobs:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
```

Comment:
When you want to use a different toolchain other than the current stable, you are supposed to get it by: `dtolnay/rust-toolchain@1.81.0`.

_The selection of Rust toolchain is made based on the particular @rev of this Action being requested. For example "dtolnay/rust-toolchain@nightly" pulls in the nightly Rust toolchain, while "dtolnay/rust-toolchain@1.42.0" pulls in 1.42.0._

_Important: the default is to match the @ rev as described above. When passing an explicit toolchain as an input instead of @ rev, you'll want to use "dtolnay/rust-toolchain@master" as the revision of the action._

Reply:
Welp, it still works. If they want to enforce that, they should prevent the other way from working :D